lib/src/rules/member_ordering.dart

```diff
@@ -202,6 +244,9 @@ class _MembersGroup {
   static const angularOutputs = _MembersGroup._('angular_outputs');
   static const angularHostBindings = _MembersGroup._('angular_host_bindings');
   static const angularHostListeners = _MembersGroup._('angular_host_listeners');
+  static const angularViewChild = _MembersGroup._('angular_view_children');
+  static const angularContentChild =
+      _MembersGroup._('angular_content_children');
 
   static List<_MembersGroup> groupsOrder = [
```

Comment:
@incendial can we make private this variable ?

Reply:
Fixed